lib/predictions/predictions.ex

```diff
@@ -80,25 +83,27 @@ defmodule Predictions.Predictions do
          else: nil
 
     seconds_until_passthrough =
-      if stop_time_update["passthrough_time"],
-        do: stop_time_update["passthrough_time"] - current_time_seconds,
+      if not revenue_trip? or schedule_relationship == :skipped,
```

Comment:
Flagging for review: This check against `schedule_relationship` is new logic since the first attempt at migrating